### DISCUSSION TITLE
Quickfix list tool

### DISCUSSION THREAD

**Discussion by gleachkr** [olimorris/codecompanion.nvim#441] [Ideas]
It would be neat if there were a tool that let codecompanion populate the quickfix list, so that you could, for example, quickly navigate from one suggested change to another

**Comment by olimorris** [comment]
That's a very nice idea.

I plan on adding a workspace feature in the near future that would create a map of the repository (similar to Aider). Letting the LLM analyze a code base, make suggestions and then parse the responses and add to the quickfix would be very cool!
